applications/kalpana-stampede3/app.json

```diff
@@ -0,0 +1,298 @@
+					"arg": "--var zeta_max",
+					"notes": {
+						"isHidden": false
+					}
+				},
+				{
+					"name": "Contour Levels",
+					"description": "Contour level specification, in the form min max step. Flag: --levels Value: three numbers, separated by spaces",
+					"inputMode": "REQUIRED",
+					"arg": "--levels z0 z1 dz",
+					"notes": {
+						"isHidden": false
+					}
+				},
+				{
+					"name": "Contour Type",
+					"description": "Contour type, polyline or polygon. Flag: --conType Value: polyline or polygon",
+					"inputMode": "REQUIRED",
+					"arg": null,
+					"notes": {
+						"enum_values": [
+							{
+								"--conType polygon": "polygon"
+							},
+							{
+								"--conType polyline": "polyline"
+							}
+						],
+						"isHidden": false
+					}
+				},
+				{
+					"name": "Path Out",
+					"description": "Path to output, *.shp or *.gpkg. Flag: --pathOut Value: file path",
+					"inputMode": "REQUIRED",
+					"arg": "--pathOut \\$_tapisExecSystemOutputDir/contours.shp",
+					"notes": {
+						"isHidden": true
+					}
+				},
+				{
+					"name": "EPSG Out",
+					"description": "EPSG coordinate system for output, 4326 recommended. Flag: --epsgOut Value: integer",
+					"inputMode": "REQUIRED",
+					"arg": "--epsgOut 4326",
+					"notes": {
+						"isHidden": true
+					}
+				},
+				{
+					"name": "Vertical Units Out",
+					"description": "Units out. Default is m. Flag: --vUnitOut Value: m or ft",
+					"inputMode": "INCLUDE_ON_DEMAND",
+					"arg": null,
+					"notes": {
+						"enum_values": [
+							{
+								"--vUnitOut m": "m"
+							},
+							{
+								"--vUnitOut ft": "ft"
+							}
+						],
+						"isHidden": false
+					}
+				},
+				{
+					"name": "Vertical Units In",
+					"description": "Units in. Default is m. Flag: --vUnitIn Value: m or ft",
+					"inputMode": "INCLUDE_ON_DEMAND",
+					"arg": null,
+					"notes": {
+						"enum_values": [
+							{
+								"--vUnitIn m": "m"
+							},
+							{
+								"--vUnitIn ft": "ft"
+							}
+						]
+					}
+				},
+				{
+					"name": "EPSG In",
+					"description": "EPSG coordinate system for input. Default is 4326. Flag: --epsgIn Value: integer",
+					"inputMode": "INCLUDE_ON_DEMAND",
+					"arg": null,
+					"notes": {
+						"isHidden": true
+					}
+				},
+				{
+					"name": "Subdomain",
+					"description": "If a subdomain is desired, full path of the subdomain polygon kml or shapefile, or list of bounding box coordinates separated by spaces W N E S. Default is None. Flag: --subDomain Value: file path or four numbers, separated by spaces",
+					"inputMode": "INCLUDE_ON_DEMAND",
+					"arg": null,
+					"notes": {
+						"isHidden": true
+					}
+				},
+				{
+					"name": "EPSG of Subdomain",
+					"description": "EPSG coordinate system for subdomain. Default is None. Flag: --epsgSubDom Value: integer",
+					"inputMode": "INCLUDE_ON_DEMAND",
+					"arg": null,
+					"notes": {
+						"isHidden": true
+					}
```

Comment:
If the input is `null` and hidden, but also not required, we can probably take it out. Unless the plan is to start letting users change this variable.  There's a couple of inputs that are like this, and taking them out might make the file shorter or easier to read.

Reply:
@rstijerina brings up a good point about these arguments being positional in the function, but should maybe be env variables instead.